src/lib/operators/aggregate_hash.cpp

```diff
@@ -418,6 +418,9 @@ void AggregateHash::_aggregate() {
               _aggregate_segment<ColumnDataType, AggregateFunction::StandardDeviationSample, AggregateKey>(
                   chunk_id, column_index, *base_segment, keys_per_chunk);
               break;
+            case AggregateFunction::Any:
```

Comment:
Added this to lines 132..
```
else if constexpr (function == AggregateFunction::Any) {
        // Gathering the group's first value for ANY() is sufficient
        return;
      }
```

Tests no longer run. The change for the sort-based aggregate is fine however. Need to better understand the hash aggregate.

Reply:
I don't think we can break here, since (in contrast to the consecutive group values in the sorted case) the hash join receives the group-by keys in an arbitrary order. We might just skip the following groups if we simply break here. And in the aggregation itself, we already check if a value has been set already.